config/config.go

```diff
@@ -63,6 +63,7 @@ ios:
   key_type: "pem" # could be pem, p12 or p8 type
   password: "" # certificate password, default as empty string.
   production: false
+  max_concurrent_pushes: 100
```

Comment:
@McPo You also need to update this config.

Reply:
with the comment?

Reply:
Yes.

Reply:
CI will check the config content between `config/testdata/config.yml` and readme

Reply:
See https://cloud.drone.io/appleboy/gorush/326/1/6